pkg/controller/certificaterequests/venafi/BUILD.bazel

```diff
@@ -46,9 +48,9 @@ go_test(
         "//pkg/apis/meta/v1:go_default_library",
         "//pkg/controller/certificaterequests:go_default_library",
         "//pkg/controller/test:go_default_library",
-        "//pkg/internal/venafi:go_default_library",
-        "//pkg/internal/venafi/api:go_default_library",
-        "//pkg/internal/venafi/fake:go_default_library",
+        "//pkg/issuer/venafi/client:go_default_library",
+        "//pkg/issuer/venafi/client/api:go_default_library",
+        "//pkg/issuer/venafi/client/fake:go_default_library",
```

Comment:
Not a big issue because we'll change this up soon anyway with @JoshVanL's work, but we're trying to remove the `pkg/issuer` directory altogether. We need to have a bigger discussion around package layout and use of `internal/` before we cut the v1.0.0 tag however, so let's not hold this up on that 😄

Reply:
> We need to have a bigger discussion around package layout and use of internal/ before we cut the v1.0.0 tag however

big +1 on that